src/module/system/action-macros/basic/escape.ts

```diff
@@ -86,7 +86,7 @@ function escapeCheckContext<ItemType extends ItemPF2e<ActorPF2e>>(
     );
 
     if (highest) {
-        const { checkType, stat: slug, subtitle } = ActionMacroHelpers.resolveStat(highest.statistic.slug);
+        const { checkType, stat: slug, subtitle } = ActionMacroHelpers.resolveStat(highest.statistic.slug, opts.actor);
```

Comment:
not sure how this is related?

Reply:
In order to get access to the skill label, I added the actor as an argument to `ActionMacroHelpers.resolveStat()`.  This is the one other call site of this function.